src/main/java/net/minecraftforge/common/BiomeManager.java

```diff
@@ -36,14 +37,8 @@
 {
     private static TrackedList<BiomeEntry>[] biomes = setupBiomes();
 
-    public static List<Biome> oceanBiomes = new ArrayList<Biome>();
```

Comment:
This is a binary breaking change, as this field was public

Reply:
`oceanBiomes` was never actually called externally in 1.13+, so keeping it private would be ok in this case. 

I made the ocean biome entry system similar to the normal biome entry system, which is why I kept it private. 

Since `oceanBiomes` was never actually used truly in 1.13+, I thought it would be okay to convert to a system more like the normal biome entry system.

Making it public would allow people to set a new value on the field also.

If you still think I should make it public, that's fine, I just wanted to give my reasoning.

Reply:
ichttt's point is that once there is an RB binary breaking changes are not allowed. So if some modder somewhere was calling `oceanBiomes` (even if it wasn't actually doing anything), it would cause their mod to start crashing.

Reply:
I see, point taken. I will make it public 👍

Although again I will say outside of this PR, in 1.13+ versions of forge, manipulating this field in any way will not affect anything in-game, so I'd be surprised if anyone would still be using it in 1.15

Reply:
Should we not mark this as @Deprecated then?

Reply:
marking the field as `@Deprecated` seems like the best course of action

Reply:
Yes, `@Deprecated` does seem like the best decision.